CHANGELOG

```diff
@@ -96,7 +96,10 @@ v3.9.2 (XXXX-XX-XX)
 
 * Added an IO heartbeat which checks that the underlying volume is writable with
   reasonable performance. The test is done every 15 seconds and can be
-  explicitly switched off. New metrics to give visibility if the test fails.
+  explicitly switched off. New metrics to give visibility if the test fails:
+    - `arangodb_ioheartbeat_delays_total`: total number of delayed io heartbeats
+    - `arangodb_ioheartbeat_duration`: histogram of execution times [us]
+    - `arangodb_ioheartbeat_failures_total`: total number of failures
```

Comment:
Probably we should add here that these metrics will only be populated if the option `--database.io-heartbeat` is set to `true`.

Reply:
Good point. Will do.